types/microsoft-ajax/index.d.ts

```diff
@@ -2505,8 +2505,7 @@ declare namespace Sys {
         * @see {@link http://msdn.microsoft.com/en-us/library/bb383800(v=vs.100).aspx}
         */
         class ProfileService {
-
-            new(): ProfileService;
+            constructor();
```

Comment:
This one can just be removed, I think.

Reply:
Well.. yes.. it is useless. 

But the [linked documentation](https://docs.microsoft.com/en-us/previous-versions/bb907039%28v%3dvs.100%29) states that the constructor exists, even though it is kind of useless.  
So I decided to leave it in.

Reply:
Sure, but a parameterless constructor is implied for classes that don’t specify a different construct signature. It’s not hurting anything, it’s just a no-op.